src/main/java/mate.academy/Main.java

```diff
@@ -4,14 +4,16 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
-        List<Product> products = productService.getAllFromFile("products.txt");
+        ProductService productService =
+                (ProductService) injector.getInstance(ProductServiceImpl.class);
+        List<Product> products = productService
+                .getAllFromFile("src/main/resources/products.txt");
```

Comment:
Verify that the `getAllFromFile` method in `ProductService` is correctly implemented and that the file path `"src/main/resources/products.txt"` is correct and accessible. This is crucial for the method to retrieve products successfully.